tests/formats/autodetect/path.in.log

```diff
@@ -0,0 +1,8 @@
+#separator \x09
+#set_separator	,
+#empty_field	(empty)
+#unset_field	-
+#path	conn
+#fields	_path	foo
+#types	string	string
+conn	yes
```

Comment:
To really test that this is working right, we should use a different value than the `#path` directive here...

Reply:
done